test/let_value_test.cpp

```diff
@@ -45,6 +45,34 @@ constexpr auto asyncVector = [](auto& context) {
         return std::vector<int>{1, 2, 3, 4};
     });
 };
+
+namespace _never_block {
+template <typename... Values>
+struct sender {
+  template <
+      template <typename...> class Variant,
+      template <typename...> class Tuple>
+  using value_types = Variant<Tuple<Values...>>;
+
+  template <template <typename...> class Variant>
+  using error_types = Variant<std::exception_ptr>;
+
+  static constexpr bool sends_done = false;
+
+  friend constexpr auto tag_invoke(tag_t<blocking>, const sender&) noexcept {
+    return blocking_kind::never;
+  }
+};
+
+inline const struct _fn {
+  template <typename... Values>
+  constexpr auto operator()(Values&&... values) const noexcept {
+    return _never_block::sender{(Values&&)values...};
+  }
+
+} never_block{};
+} // namespace _never_block
+using _never_block::never_block;
 } // anonymous namespace
```

Comment:
Can you add a test case with this _Sender_, please?  I kind of expect the `value_types = Variant<Tuple<int>, Tuple<double>>` to trigger new, possibly broken code.

```suggestion
namespace _multi {
struct _multi_sender {
  template <
      template <typename...> class Variant,
      template <typename...> class Tuple>
  using value_types = Variant<Tuple<int>, Tuple<double>>;

  template <template <typename...> class Variant>
  using error_types = Variant<std::exception_ptr>;

  static constexpr bool sends_done = false;
};

inline const struct _fn {
  template <typename... Values>
  constexpr auto operator()(Values&&...) const noexcept {
    return _multi::_multi_sender{};
  }
} multi_sender{};
} // namespace _multi

using _multi::multi_sender;

} // anonymous namespace
```